gh-labels.rst

```diff
@@ -0,0 +1,169 @@
+
+interpreter-core
+    Used for issues in interpreter core (``Objects/``, ``Python/``,
+    ``Grammar/``, and ``Parser/`` dirs).
+
+extension-modules
+    Used for issues involving C modules in the ``Modules/`` dir.
+
+tests
+    Used for issues involving only Python's regression test suite, i.e.
+    files in the ``Lib/test/`` dir.
+
+Other
+^^^^^
+
+new
+    Denotes that the issue hasn't been looked at by triagers or core
+    developers yet.
+
+easy
+    Denotes that the issue is a good candidate for a newcomer to address.
+
+
+Labels specific to PRs
+----------------------
+
+DO-NOT-MERGE
+    Used on PRs to prevent miss-islington from being able
+    to automatically merge the pull request. This label is appropriate when a PR
+    has a non-trivial conflict with the branch it is being merged into.
+
+expert-asyncio
+    Used for PRs which involve changes to the asyncio module
+    or other asynchronous frameworks that utilize it.
+
+invalid
+    Used manually for PRs that do not meet basic requirements and
+    automatically added by bedevere when PR authors attempt to merge maintenace
+    branches into the main branch. During events such as the October
+    Hacktoberfest, this label will prevent the PR from counting toward the
+    author's contributions.
+
+needs backport to X.Y
+    Used for PRs which are appropriate to backport to
+    branches prior to main. Generally, backports to the maintenance branches
+    are primarily bugfixes and documentation clarifications. Backports to the
+    security branches are strictly reserved for PRs involving security fixes, such as
+    crashes, privilege escalation, and DoS. The use of this label will cause
+    miss-islington to attempt to automatically merge the PR into the branches
+    specified.
+
+skip issue
+    Used for PRs which involve trivial changes, such as typo fixes,
+    comment changes, and section rephrases. The majority of PRs require
+    an issue to be attached to, but if there are no code changes and the
+    section being modified retains the same meaning, this label might be
+    appropriate.
+
+skip news
+    Similar to the skip issue label, this label is used for PRs which
+    involve trivial changes, backports, or already have a relevant news entry
+    in another PR. Any potentially impactful changes should have a
+    corresponding news entry, but for trivial changes it's commonly at the
+    discretion of the PR author if they wish to opt-out of making one.
+
+sprint
+    Used for PRs authored during an in-person sprint, such as
+    at PyCon, EuroPython, or other official Python events. The label is
+    used to prioritize the review of those PRs during the sprint.
+
+stale
+    Used for PRs that include changes which are no longer relevant or when the
+    author hasn't responded to feedback in a long period of time. This label
+    helps core developers quickly identify PRs that are candidates for closure
+    or require a ping to the author.
```

Comment:
I think stale is just simply no activity, not awaiting changes. No activity might include waiting for an initial review (or a wontfix, as the case may be).